server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -156,13 +161,27 @@ impl<'a> RequisitionRowRepository<'a> {
     }
 
     #[cfg(not(feature = "postgres"))]
-    pub fn upsert_one(&self, row: &RequisitionRow) -> Result<(), RepositoryError> {
+    fn upsert_one_etc(&self, row: &RequisitionRow) -> Result<(), RepositoryError> {
         diesel::replace_into(requisition_dsl::requisition)
             .values(row)
             .execute(&self.connection.connection)?;
         Ok(())
     }
 
+    fn toggle_is_sync_update(&self, id: &str, is_sync_update: bool) -> Result<(), RepositoryError> {
+        diesel::update(requistion_is_sync_update::table.find(id))
+            .set(requistion_is_sync_update::dsl::is_sync_update.eq(is_sync_update))
+            .execute(&self.connection.connection)?;
+
+        Ok(())
+    }
+
+    pub fn upsert_one(&self, row: &RequisitionRow) -> Result<(), RepositoryError> {
```

Comment:
ah right. surely this fn should be called the `_etc` version?

Reply:
oops, as per pervious answer